### DISCUSSION TITLE
vercel webhook Trigger builds based on code directory？

### DISCUSSION THREAD

**Discussion by DreamPWJ** [vercel/community#470] [Help] [answered]
The deployment code is under a specific directory. I hope that the build will only be triggered when the relevant code is submitted.

**Comment by swarnava** [comment]
Hi there,

I am assuming you are using [Deploy Hook](https://vercel.com/docs/concepts/git/deploy-hooks) to setup your deployment. In this case, it really depending on how your project is setup. In general, the Deploy Hook get triggered whenever there is a GET or POST request received to the provided URL.

**Comment by wiiiimm** [comment, marked as answer]
@DreamPWJ There is a "Ignored Build Step" settings you can set for your project. In there, you can configure scripts that can be used to determine if a commit / PR needs to be deployed. You can find more information over at our support article - [How do I use the "Ignored Build Step" field on Vercel?](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel).

I think this [example](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel?query=ignored%20build%20s#with-folders-and-workspaces) is what you can follow:
> ... By using this command, Vercel will only build deployments when changes are made inside of the packages/frontend/ directory. If the folder ./packages/frontend/ is your selected "Root Directory", then you can use:
